xbmc/utils/StreamDetails.cpp

```diff
@@ -282,7 +282,7 @@ std::string CStreamDetails::GetVideoLanguage(int idx) const
   const CStreamDetailVideo* item =
       dynamic_cast<const CStreamDetailVideo*>(GetNthStream(CStreamDetail::VIDEO, idx));
   if (item)
-    return item->m_strLanguage;
+    return item->m_strLanguage.empty() ? "und" : item->m_strLanguage;
```

Comment:
Instead of using the string literal can you move this to the anonymous namespace as a `const char * LANGUAGE_UNDEFINED = "und"`; and just use `LANGUAGE_UNDEFINED`?